data/phones/postprocess

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+# Runs summary generation script.
+
+set -eou pipefail
+
+main() {
```

Comment:
If this is just one line I think you don't have to use a `main`. I just use it when there are multiple steps.